collect_app/src/androidTest/assets/all-widgets.xml

```diff
@@ -263,24 +401,37 @@
       <bind nodeset="/all-widgets/select_one_widgets/grid_widget_compact2" type="select1"/>
       <bind nodeset="/all-widgets/select_one_widgets/grid_widget_quickcompact" type="select1"/>
       <bind nodeset="/all-widgets/select_one_widgets/grid_widget_quickcompact2" type="select1"/>
+      <bind nodeset="/all-widgets/select_one_widgets/select_one_image_map" type="select1"/>
       <bind nodeset="/all-widgets/select_multi_widgets/select_multi_widget" type="select"/>
       <bind nodeset="/all-widgets/select_multi_widgets/grid_multi_widget_compact" type="select"/>
       <bind nodeset="/all-widgets/select_multi_widgets/grid_multi_widget_compact2" type="select"/>
       <bind nodeset="/all-widgets/select_multi_widgets/spinner_multi_widget" type="select"/>
+      <bind nodeset="/all-widgets/select_multi_widgets/select_multi_image_map" type="select"/>
       <bind nodeset="/all-widgets/table_list_test/table_list_test_label" type="select1"/>
       <bind nodeset="/all-widgets/table_list_test/table_list_test_label_2" type="select"/>
       <bind nodeset="/all-widgets/table_list_test/table_list_1" type="select1"/>
       <bind nodeset="/all-widgets/table_list_test/table_list_2" type="select"/>
       <bind nodeset="/all-widgets/table_list_test/list_widget" type="select1"/>
       <bind nodeset="/all-widgets/table_list_test/list_multi_widget" type="select"/>
-      <bind nodeset="/all-widgets/my_trigger"/>
       <bind calculate="concat('uuid:', uuid())" nodeset="/all-widgets/meta/instanceID" readonly="true()" type="string"/>
     </model>
   </h:head>
   <h:body>
     <input ref="/all-widgets/intro">
-      <label> This form showcases the different available question types (_widgets_).</label>
-      <hint>This is a sample for introductory and debugging purposes. It is updated periodically so check "Get Blank Form" to get the latest! Version 2017121301 adds:
+      <label>Welcome to ODK Collect! This form showcases the different available question types (_widgets_).</label>
+      <hint>This is a sample for introductory and debugging purposes. It is updated periodically so check "Get Blank Form" to get the latest! 
+
+Version 2018051401 adds:
+&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;- File widget
+
+Version 2018032801 adds:
+&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;- Appearance new-front (selfie) for video type
+
+Version 2018022001 adds:
+&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;- Appearance image-map for select\_one and select\_multiple types
+&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;- Appearance new for image, audio, video types
+
+Version 2017121301 added:
```

Comment:
Do we still need to state what the previous version added?

Reply:
Yes @xsteelej, generally just the features that the latest version has added are specified. But then, we need to make the required changes in the form on the server, right?

Reply:
Btw thanks for reviewing, @xsteelej...:)

Reply:
No problem @SaumiaSinghal